Core/Inc/util.h

```diff
@@ -18,5 +18,6 @@ bool get_individual_flag(osEventFlagsId_t ef_id, uint32_t flag);
 
 #define sign(x) (x >= 0) - (x < 0)
 #define max(x,y) (x > y) ? x : y
+#define min(x,y) (x < y) ? x : y
```

Comment:
encapsula essa e a de cima em parênteses, é bom fazer isso em defines pra evitar comportamento inesperado por causa de precedência de operadores

Reply:
seria tipo assim, se vc tiver em dúvida:
```suggestion
#define min(x,y) (((x) < (y)) ? (x) : (y))
```